src/modules/user/commands/create-user/create-user.service.ts

```diff
@@ -11,7 +11,10 @@ import { UserEntity } from '../../domain/entities/user.entity';
 import { UserAlreadyExistsError } from '../../errors/user.errors';
 
 @CommandHandler(CreateUserCommand)
-export class CreateUserService extends CommandHandlerBase {
+export class CreateUserService extends CommandHandlerBase<
+  ID,
+  UserAlreadyExistsError
+> {
```

Comment:
I added this to that I get typing support while accessing this method through the e2e test (internal interface). As this is not needed anymore, I can also revert these changes.
Why duplicate the  `execute` method? Because it wraps everything into a valid database transaction.